packages/near-lake-primitives/src/types/receipts.ts

```diff
@@ -80,6 +161,9 @@ class DeleteAccount {
   constructor(readonly beneficiaryId: string) { }
 };
 
+/**
+ * A representation of the original `ActionView` from `near-primitives`.
```

Comment:
In many places in this PR, we refer to `near-primitives` which may be confusing to the user because it may seem like we are referring to this exact same library as it has got the same name. 

Trying to figure out how we can fix this. Perhaps add a link to the actual `near-primitives` library or make it obvious somehow that we are referring to the `core` lib types?

Reply:
Added links to the [nearcore](https://github.com/near/nearcore/tree/master/core/primitives) repo for clarity and desambiguation